test/tzdata/version.jl

```diff
@@ -1,6 +1,12 @@
 using TimeZones.TZData: ARCHIVE_DIR, TZDATA_VERSION_REGEX, TZDATA_NEWS_REGEX
 using TimeZones.TZData: read_news, extract, tzdata_version_dir, tzdata_version_archive
-using TimeZones.TZData: active_version, active_archive
+using TimeZones.TZData: active_version, active_archive, active_dir
+if VERSION >= v"1.4"
+    using Pkg.Artifacts
+    using Pkg.Artifacts: artifacts_dirs
+else
+    import TimeZones.TZData: @artifact_str
```

Comment:
Unused

Reply:
I just realied, if we'll use the `use_artifacts = VERSION >= v"1.4"` will this go well along with `@static` or not? Because if not, then the `@artifact_str` will be missing again.

Reply:
`@static` at the top-level I believe isn't necessary so it can be safely left out. Also note that `@static` can be applied to any `if` with a condition that can be resolved at compile time